@loaders.gl/shapefile compile issue Can't resolve 'fs'

I'm trying to compile my JS using laravel-mix and I keep getting this error:

```
ERROR in ./node_modules/@loaders.gl/shapefile/dist/esm/lib/filesystems/node-filesystem.js
Module not found: Error: Can't resolve 'fs' in '/home/forge/**************/node_modules/@loaders.gl/shapefile/dist/esm/lib/filesystems'
```

I though I simply missing the fs module, so I ran `npm install fs` but that didn't help, any help would be appreciated.

--- a/modules/shapefile/package.json
+++ b/modules/shapefile/package.json
@@ -21,6 +21,10 @@
   "module": "dist/esm/index.js",
   "esnext": "dist/es6/index.js",
   "sideEffects": false,
+  "browser": {
+    "./src/lib/filesystems/node-filesystem.js": false,
+    "fs": false
+  },
   "files": [
     "src",
     "dist",

